src/CoverageTools.jl

```diff
@@ -267,51 +267,60 @@ function process_folder(folder="src")
     return source_files
 end
 
+# matches julia allocation files with and without the PID
+ismemfile(filename) = occursin(r"\.jl\.?[0-9]*\.mem$", filename)
+# matches an allocation file for the given sourcefile. They can be full paths
+# with directories, but the directories must match
+function ismemfile(filename, sourcefile)
+    startswith(filename, sourcefile) || return false
+    ismemfile(filename)
+end
+
 # matches julia coverage files with and without the PID
 iscovfile(filename) = occursin(r"\.jl\.?[0-9]*\.cov$", filename)
 # matches a coverage file for the given sourcefile. They can be full paths
 # with directories, but the directories must match
 function iscovfile(filename, sourcefile)
     startswith(filename, sourcefile) || return false
-    occursin(r"\.jl\.?[0-9]*\.cov$", filename)
+    iscovfile(filename)
 end
 
 """
-    clean_folder(folder::AbstractString)
+    clean_folder(folder::AbstractString; include_memfiles::Bool=false)
```

Comment:
Maybe `include_memfiles` is not the best name (`also_memfiles`? also not great)

Well naming is hard. I think it is good enough 🤷